packages/react-native-editor/package.json

```diff
@@ -64,7 +64,7 @@
   },
   "scripts": {
     "start": "react-native start",
-    "start:reset": "npm run clean:runtime && npm run start --reset-cache",
+    "start:reset": "npm run clean:runtime && npm run start -- --reset-cache",
```

Comment:
Are you sure that's required? It's not required for my version of `npm` (6.14.2), maybe you have an older version?

Reply:
It's correct, it's how you should pass CLI options to another npm scripts.

Reply:
It still starts fine without the `--` but the argument isn't passed so you shouldn't see `warning: the transform cache was reset.`